### PR TITLE
Bank page hero refresh

### PR DESCRIPTION
**Pull request by exu3**
To do:
- [x] Change Bank og-image
- [x] Change Bank hero image
- [x]

### CODE REVIEW

**Review by sampoder — COMMENTED** [review]
We could probably do with a higher quality image, there is a large version of this image (20MB ish, so it will probably need compressing or `next/image` use) and a couple others here: https://drive.google.com/drive/folders/1rmrTzriJ2STsGNh7zV3uQwRkfN1VJZVO?usp=sharing

**Comment by exu3** [comment]
Thanks for linking the Drive folder; the photos look really nice. Thoughts on the new image?

Screen Shot 2022-05-03 at 21 11 44

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/bank/\[type\]**](https://site-o1o93xkzx.hackclub.dev/bank/[type]) |
|:-:|
|<a href="https://site-o1o93xkzx.hackclub.dev/bank/[type]">Screenshot of /bank/[type]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/PqcYRK0ZE87yOHyAu4apkaCVQCGQmR4c7qiBvK-sFmT0fK6QjyKDgHF5XLBVvxWLegIED0D4n8c0hEjLy3aV8A.png">(view full size)</a> |







Commit 83ee54ff867af95a458ed6011a4e6563bd0dcb68 (https://site-o1o93xkzx.hackclub.dev).

File: components/bank/landing.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { Box, Button, Heading, Link, Text, Container, Badge } from 'theme-ui'
 import Fade from 'react-reveal/Fade'
 import ScrollHint from './scroll-hint'
+import Image from 'next/image'
+import { hero } from '../../public/bank/bg.webp'
```
Comment (sampoder) [inline components/bank/landing.js]:
```suggestion
import hero from '../../public/bank/bg.webp'
```

Comment (exu3) [reply components/bank/landing.js]: Ah thanks

File: components/bank/landing.js [hunk 2]
```diff
@@ -110,7 +112,7 @@ export default function Landing({ showButton, eventsCount }) {
               fontWeight: 'normal'
             }}
           >
-            Tuscon, AZ
+            Singapore 🇸🇬
```
Comment (sampoder) [inline components/bank/landing.js]: Flag emojis don't work on Windows :(

```suggestion
            Singapore
```

**Review by sampoder — APPROVED** [review]
🎉 🙌 🌈

**Pull request merged**